src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/ModellingGroupController.kt

```diff
@@ -109,6 +118,18 @@ open class ModellingGroupController(context: ControllerContext)
         return data.responsibilities
     }
 
+    fun getTouchstones(context: ActionContext, repo: TouchstoneRepository): List<Touchstone>
+    {
+        val groupId = groupId(context)
+
+        var touchstones = repo.getTouchstonesByGroupId(groupId)
+        if (!context.hasPermission(touchstonePreparer))
```

Comment:
This logic is duplicated from the `TouchstoneController`, so would be nice to pull it out into a shared method, maybe in `CommonSecurityRules`